Source/Core/AttributeCompression.js

```diff
@@ -23,21 +25,22 @@ define([
     var AttributeCompression = {};
 
     /**
-     * Encodes a normalized vector into 2 SNORM values in the range of [0-255] following the 'oct' encoding.
+     * Encodes a normalized vector into 2 SNORM values in the range of [0-rangeMax] following the 'oct' encoding.
      *
      * Oct encoding is a compact representation of unit length vectors.  The encoding and decoding functions are low cost, and represent the normalized vector within 1 degree of error.
      * The 'oct' encoding is described in "A Survey of Efficient Representations of Independent Unit Vectors",
      * Cigolle et al 2014: {@link http://jcgt.org/published/0003/02/01/}
      *
      * @param {Cartesian3} vector The normalized vector to be compressed into 2 byte 'oct' encoding.
      * @param {Cartesian2} result The 2 byte oct-encoded unit length vector.
+     * @param {Number} [rangeMax=255] The maximum value of the SNORM range, 255 by default.
      * @returns {Cartesian2} The 2 byte oct-encoded unit length vector.
      *
      * @exception {DeveloperError} vector must be normalized.
      *
      * @see AttributeCompression.octDecode
      */
-    AttributeCompression.octEncode = function(vector, result) {
+    AttributeCompression.octEncode = function(vector, result, rangeMax) {
```

Comment:
`result` parameters should come last so we need to introduce a new function (and we can implement this function using that new one; no need to deprecate this).